### ISSUE TITLE
Disclaimer link should not open in new tab

### ISSUE THREAD

**Issue by emilyjablonski** [bloom-housing/bloom#4359]
Currently in the footer, the privacy policy link does not open in a new tab, but the disclaimer link does.

The experience should be consistent, and according to [accessibility guidelines](https://www.w3.org/TR/WCAG20-TECHS/G200.html), we should minimize opening a new tab in most cases, so neither should open in a new tab.
